index.bs

```diff
@@ -841,6 +841,31 @@ that authors can use to extend the language
 without breaking future native functionality,
 to reduce such dilemmas in the future.
 
+<h3 id="text-based-syntaxes">Design text-based syntax with humans in mind</h3>
+
+When designing a text-based syntax, consider human usability and readability,
+even when you expect the syntax to be primarily generated by tooling.
+
+Historically, every syntax that could be read by humans, inevitably ended up being authored by humans too.
+Even when syntax is primarily authored by tools,
+it is still effectively authored by humans: the tool developers.
+Therefore, optimizing it for human usability also makes it easier to develop tools to generate it.
+
+Human usability may often be at odds with parsing performance and filesize efficiency.
+Ideally, the syntax would be optimized for human usability, and tooling would optimize the syntax for efficiency.
```

Comment:
I was a little unclear as to what this line was trying to say; @LeaVerou explained it's about situations e.g. where you can minify a file with tooling to make it more efficient - that was good clarification I think, so I suggest including that in the text.

It feels like there could be quite big implications here (accepting tooling as integral to the process, if we've not already explicitly done so elsewhere). Solving for both human readability and efficiency is an important challenge.